src/components/frame/v5/pages/UserCryptoToFiatPage/partials/ContactDetailsForm/CountrySelect.tsx

```diff
@@ -25,11 +25,21 @@ export const CountrySelect = () => {
     country: item,
   }));
 
-  const { setValue } = useFormContext();
+  const {
+    setValue,
+    trigger,
+    formState: { isSubmitted },
+  } = useFormContext();
 
   const handleSelect = () => {
     // if country changed user should choose state of new country
     setValue('state', '');
+
+    // If no attempt to submit the form has been made yet, do not trigger state validation
+    if (isSubmitted) {
```

Comment:
I think here the condition should be ` if (!isSubmitted) {` otherwise I ran into this situation
No country is selected
<img width="600" alt="Screenshot 2024-08-08 at 10 53 16" src="https://github.com/user-attachments/assets/8004d516-8480-4f15-8587-45cae0876fce">
Select country -> State validation is already triggered
<img width="562" alt="Screenshot 2024-08-08 at 10 53 21" src="https://github.com/user-attachments/assets/0598ef6d-8888-4439-9787-2e56a229f4b0">
Error is still shown after selecting a value for the state
<img width="543" alt="Screenshot 2024-08-08 at 10 53 29" src="https://github.com/user-attachments/assets/e2fa25c6-31f7-4f6e-a061-0b88d62a5455">

Reply:
🤦‍♂️ Yes of course